### PR TITLE
Control Structure - change presentation and add exe

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

**Comment by shiramax** [comment]
@sleviim please review :)

File: docs/course_materials/control_structures.html [hunk 1]
```diff
@@ -266,45 +393,18 @@ <h3><b>For loop on a list</b></h3>
...
-# Start from 10 until 0, decreased by 1
-# (10,9,8,7,6,5,4,3,2,1)
-for x in range(10,0,-1):
-    print(x)
-                    </code></pre>
+                  <h3><b>Let's Practice for loop together!</b></h3>
+                  <ul>
+                    <li>The name of the file should be <b>sum.py</b></li>
+                    <li>Accept number from user and calculate the sum of all number between 1 and given number
+                      For example user given 10 so the output should be 55
```
Comment (sleviim) [inline docs/course_materials/control_structures.html:400]: For example, for receiving 10, the output should be 55.

File: docs/course_materials/control_structures.html [hunk 2]
```diff
@@ -266,45 +393,18 @@ <h3><b>For loop on a list</b></h3>
...
+                    </li>
+                  </ul>
+                  <img data-natural-width="340" data-natural-height="234" style="" data-src="https://posteo.de/images/daten.png" />
                 </section>
+
                 <section>
-                    <h3><b>Nested Loop</b></h3>
+                    <h3><b>Extra - Nested Loop</b></h3>
```
Comment (sleviim) [inline docs/course_materials/control_structures.html:407]: What about adding a slide with 'Extra' before that?

**Review by sleviim — CHANGES_REQUESTED** [review]

File: docs/course_materials/control_structures.html [hunk 3]
```diff
@@ -227,14 +257,111 @@ <h3><b>While with condition</b></h3>
...
+      for i in ("a",2,3,True,5):
+          print(i)
+
+      a
+      2
+      3
+      True
+      5
+
+                  </code></pre>
```
[the remark this answers is not in the document]
Comment (shiramax) [reply docs/course_materials/control_structures.html:324]: we have a loop of strings in line 387, and I would like to show the students we can do all options.

**Review by sleviim — APPROVED** [review]
LGTM

**Comment by rollandf** [comment]
:100:

**Pull request merged**